app/assets/javascripts/journal_entry.js

```diff
@@ -204,6 +209,18 @@ function load_name_status() {
   });
 }
 
+function load_btn_status() {
+  if((is_current_period() == false) ||
+    ($(".log_book option:selected").attr("open-balance") == "true")) {
+    $(".btn-delete, .btn-save").attr("disabled", true);
+  }else if(is_transaction_exist() == false){
+    $(".btn-delete").attr("disabled", true);
+    $(".btn-save").attr("disabled", false);
+  }else{
+    $(".btn-delete, .btn-save").attr("disabled", false);
+  }
+}
+
```

Comment:
can we write like this: 
function load_btn_status() {
  $(".btn-delete, .btn-save").attr("disabled", false);
  if((is_current_period() == false) ||
    ($(".log_book option:selected").attr("open-balance") == "true")) {
    $(".btn-delete, .btn-save").attr("disabled", true);
  }else if(is_transaction_exist() == false){
    $(".btn-delete").attr("disabled", true);
  }
}